src/containers/HeaderDashboard/styles.js

```diff
@@ -0,0 +1,45 @@
+import styled from "styled-components";
+
+export const Header = styled.header`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 10px 20px;
+
+  img {
+    height: 40px;
+  }
+
+  @media (max-width: 767.98px) {
+    justify-content: center;
+  }
+`;
+
+export const PhotoUser = styled.div`
```

Comment:
Please, change the `PhotoUser` to be a component and not an item of HeaderDashboard component because we'll use the `PhotoUser` in other contexts